L1Trigger/L1THGCal/plugins/ntuples/HGCalTriggerNtupleGen.cc

```diff
@@ -12,7 +12,7 @@
 #include "TrackPropagation/RungeKutta/interface/defaultRKPropagator.h"
 #include "TrackPropagation/RungeKutta/interface/RKPropagatorInS.h"
 #include "FastSimulation/Event/interface/FSimEvent.h"
-#include "FastSimulation/CaloGeometryTools/interface/Transform3DPJ.h"
+#include "FastSimulation/Particle/interface/ParticleTable.h"
```

Comment:
This merge was incorrect. The line 15 needs to go away.

Reply:
@Dr15Jones ok, both the lines need to be absent, sorry for the confusion